deployment/marketplace/digitialocean/one-click-droplet/.gitignore

```diff
@@ -0,0 +1,2 @@
+.vscode/*
```

Comment:
do we need this file?

Reply:
I have added it to the main .gitignore . This folder is used in [Visual Studio Code](https://code.visualstudio.com/) like `.idea` in JetBrains IDE.